crypto/rsa/rsa_sp800_56b_check.c

```diff
@@ -289,6 +289,11 @@ int ossl_rsa_sp800_56b_check_public(const RSA *rsa)
         return 0;
 
     nbits = BN_num_bits(rsa->n);
+    if (nbits > OPENSSL_RSA_MAX_MODULUS_BITS) {
+        ERR_raise(ERR_LIB_RSA, RSA_R_MODULUS_TOO_LARGE);
+        return 0;
+    }
```

Comment:
Should we also add a check for RSA_MIN_MODULUS_BITS (512), as in done when generating?

Reply:
IMO it should be a separate PR